plugins/kernel_v2/config/vm_trigger.rb

```diff
@@ -149,6 +157,9 @@ def finalize!
           @run_remote = new_run
         end
 
+        if @abort == true
+          @abort = 0
```

Comment:
I think that the default value should be a non-zero value here. Since we are aborting the current run, that implies that the run failed and a successful exit code should not be returned. Perhaps defaulting this value to `1` instead.